CHANGELOG.md

```diff
@@ -3,6 +3,45 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.9](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.8...v4.0.0-alpha.9) (Nov 09, 2020)
+
+### @material-ui/x-grid@v4.0.0-alpha.9 / @material-ui/data-grid@v4.0.0-alpha.9
+
+- [DataGrid] fix keyboard with multiple grids (#562) @dtassone
+- [DataGrid] Add touch support on column resize (#537) @danailH
+- [DataGrid] Refactor containerSizes in smaller state (#544) @dtassone
+- [DataGrid] Fix display of row count and selected rows on mobile (#508) @oliviertassinari
+- [DataGrid] Apply review from #412 (#515) @oliviertassinari
+- [DataGrid] Avoid paint step (#531) @oliviertassinari
+- [DataGrid] Refactor rendering, remove rafUpdate (#532) @Dtassone
+- [DataGrid] add missing reselect dep (#534) @dtassone
+- [DataGrid] Raf Timer stored in apiRef (#506) @dtassone
+- [DataGrid] Fix webpack v5 (#449) @oliviertassinari
+- [DataGrid] Rework columnReorder to work with the new state management (#505) @danailH
+- [DataGrid] Spotting perf issues (#501)  @dtassone
+- [DataGrid] refactor columns scrolling (#500) @dtassone
+- [DataGrid] Replace require with import (#455) @oliviertassinari
+- [DataGrid] Restore regression test (#503) @oliviertassinari
+- [DataGrid] Refactor state (#412) @dtassone
```

Comment:
I think we need to check what falls under Feature and what under Core because for instance there are things here that are for fixing or refactoring and maybe it makes more sense for them to be under Core?